src/schemas/v2_route.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
This is the thing meant to be edited by a human. @robinlovelace-ate , without me really describing the schema language somewhere, is this intuitive enough to figure out how to write? To me this is much more clear than all the equivalent JSON Schema examples I could come up with

Reply:
Taking a look...